nc_export.py

```diff
@@ -0,0 +1,3 @@
+from pysd.tools.ncfiles import nc_to_csv
+
+nc_to_csv("../netcdf_tests/results.nc", "results.tab")
```

Comment:
I think that is a local debugging file that should not be added

Reply:
Indeed! my bad